waku/v2/discv5/discover.go

```diff
@@ -318,7 +318,8 @@ func evaluateNode(node *enode.Node) bool {
 		return false
 	}
 
-	if !isWakuNode(node) || !hasTCPPort(node) {
+	//  TODO: consider node filtering based on ENR; we do not filter based on ENR in the first waku discv5 beta stage
+	if /*!isWakuNode(node) ||*/ !hasTCPPort(node) {
```

Comment:
I must have picked this up from `status/master` along the way. I didn't make this change.